CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
    * FIXED: Allow u-turns at no-access barriers when forced by heading [#2875](https://github.com/valhalla/valhalla/pull/2875)
 
 * **Enhancement**
+   * CHANGED: Clarify usage of directions options [#2886](https://github.com/valhalla/valhalla/pull/2886)
```

Comment:
can you move this to the bottom of this enhancement section? i know its wierd but we've been following this pattern

Reply:
looks like you deleted it and merged; I guess a small edit to the docs is not worth the changelog? I will make sure to do so next time though!

Reply:
yeah at the end of the day it didnt seem necessary, thanks!